openshift/e2e-tests-openshift.sh

```diff
@@ -324,11 +366,15 @@ function delete_istio_openshift(){
   oc delete ControlPlane/basic-install -n istio-system
 }
 
-function delete_serving_openshift() {
-  echo ">> Bringing down Serving"
-  oc delete --ignore-not-found=true -f serving-resolved.yaml
+function delete_knative_openshift() {
+  echo ">> Bringing down Knative Serving, Build and Pipeline"
+  oc delete --ignore-not-found=true -n $OLM_NAMESPACE -f knative-serving.catalogsource-ci.yaml
   oc delete --ignore-not-found=true -f third_party/config/build/release.yaml
   oc delete --ignore-not-found=true -f third_party/config/pipeline/release.yaml
+
+  oc delete project $SERVING_NAMESPACE
+  oc delete project knative-build
+  oc delete project knative-build-pipeline
 }
```

Comment:
This is pretty brute force. I've seen this hang indefinitely, but you may be fine. :)